### ISSUE TITLE
Looking for Collaborators / Maintainers for this Lib

### ISSUE THREAD

**Issue by Keyang** [Keyang/node-csvtojson#378]
TL;DR This lib will be kept maintained and supported. Anyone interested in being a collaborator please reply on this thread.

Hi All,
This library has been published since 2013 and I really enjoy working on it. Thanks for your support all the time.
I had a big career change last year and barely had time to maintain this library. Until recent, I was reminded in #377 that the last update was made 9 months ago -- how embarrassing.
I will spend some time occasionally to keep this library moving forward in future to fix critical bugs and add important features. However, it is always welcome anyone who wants to be a collaborator or maintainer on this library.
Here is the plan:
1. Please reply to this thread if you want to be a collaborator / maintainer. I will add you to access the github repo.
2. Initially, for each release, I will do a final check and deploy to npm. Eventually, the core contributors will be able to push to npm.

This is my first time to share an open source project with other developers so I would love to hear any suggestions.

Regards,
Keyang Xiang

**Comment by jeremyrajan** [comment]
More than happy to help :)

**Comment by nirlanka** [comment]
I'd love to help! ^_^

**Comment by dogabudak** [comment]
I would love to help :)

**Comment by cecilRac** [comment]
Interested in contributing

**Comment by cotterjd** [comment]
I'm interested also

**Comment by cotterjd** [comment]
I'm interested also

**Comment by markstos** [comment]
Has there been progress here with onboarding new maintainers here? It would be nice to get a new release out that removes the 7 MB `coverage/` folder. The issue was worked on some here:

https://github.com/Keyang/node-csvtojson/commit/a9de37ab5a71fede4627d1f65a9777ed62b7cf9f
